DabraWarq/DWdavies01.xml

```diff
@@ -45,740 +55,740 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+
+                  <msItem xml:id="ms_i1.3.30.1">
+                    <locus from="74ra" to="75vb"/>
+                    <title type="complete" ref="LIT3616Miracle"/>
+                    <textLang mainLang="gez"/>
+                  </msItem>
+                  <msItem xml:id="ms_i1.3.30.2">
+                    <locus target="#75vb"/>
+                    <title type="complete" ref="LIT4274AkkonuB#Stanza30"/>
+                    <textLang mainLang="gez"/>
+                  </msItem>
+                  <msItem xml:id="ms_i1.3.30.3">
+                    <locus target="#76r"/>
+                    <title type="complete" ref="LIT5673AkkonuColl#AkkMin30"/>
+                    <textLang mainLang="gez"/>
+                  </msItem>
+                </msItem>
+
+                <msItem xml:id="ms_i1.3.31">
+                  <locus from="76va" to="79r"/>
+                  <textLang mainLang="gez"/>
+                  <title type="complete" ref="LIT5673AkkonuColl#AkkColl31"/>
+
+                  <msItem xml:id="ms_i1.3.31.1">
+                    <locus from="79va" to="78ra"/>
+                    <title type="complete" ref="LIT3617Miracle"/>
+                    <textLang mainLang="gez"/>
+                  </msItem>
+                  <msItem xml:id="ms_i1.3.31.2">
+                    <locus target="#78rba"/>
+                    <title type="complete" ref="LIT4274AkkonuB#Stanza31"/>
+                    <textLang mainLang="gez"/>
+                  </msItem>
+                  <msItem xml:id="ms_i1.3.31.3">
+                    <locus from="78v" to="79r"/>
+                    <title type="complete" ref="LIT5673AkkonuColl#AkkMin31"/>
+                    <textLang mainLang="gez"/>
+                  </msItem>
+                </msItem>
+
+                <msItem xml:id="ms_i1.3.32">
+                  <locus from="79va" to="80v"/>
+                  <textLang mainLang="gez"/>
+                  <title type="complete" ref="LIT5673AkkonuColl#AkkColl32"/>
+
+                  <msItem xml:id="ms_i1.3.32.1">
+                    <locus from="79va" to="80ra"/>
+                    <title type="complete" ref="LIT3618Miracle"/>
+                    <textLang mainLang="gez"/>
+                  </msItem>
+                  <msItem xml:id="ms_i1.3.32.2">
+                    <locus target="#80rb"/>
+                    <title type="complete" ref="LIT4274AkkonuB#Stanza32"/>
+                    <textLang mainLang="gez"/>
+                  </msItem>
+                  <msItem xml:id="ms_i1.3.32.3">
+                    <locus target="#143v"/>
+                    <title type="complete" ref="LIT5673AkkonuColl#AkkMin32"/>
+                    <textLang mainLang="gez"/>
+                  </msItem>
+                  <msItem xml:id="ms_i1.3.32.4">
+                    <locus from="80rb" to="80va"/>
+                    <title type="complete" ref="LIT4274AkkonuB#Stanza32"/>
+                    <textLang mainLang="gez"/>
+                  </msItem>
```

Comment:
the nesting is still wrong. 1.3.32 most open after 1.3.31 is closed; 1.3 must close with `</msItem>` before 1.4 starts, please click on each msItem to see the nesting clearly